{
	"vulinfoid": "JVNDB-2022-010349",
	"vulinfodata": {
		"title": "Google の Google TensorFlow における入力確認に関する脆弱性",
		"vulinfodescription": {
			"overview": "Google の Google TensorFlow には、入力確認に関する脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Google",
					"productname": "Google TensorFlow",
					"cpe": {
						"text": "cpe:/a:google:tensorflow",
						"version": "2.2"
					},
					"versionnumber": [
						"2.6.4 未満",
						"2.7.0",
						"2.7.0 以上 2.7.2 未満",
						"2.8.0",
						"2.9.0"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "Low",
						"type": "Base"
					},
					"base": "2.1",
					"vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "Medium",
						"type": "Base"
					},
					"base": "5.5",
					"vector": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダアドバイザリまたはパッチ情報が公開されています。参考情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2022-29211",
					"url": "https://www.cve.org/CVERecord?id=CVE-2022-29211"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2022-29211",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-29211"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (GHSA-xrp2-fhq4-4q3w)",
					"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-xrp2-fhq4-4q3w"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (histogram_op.cc)",
					"url": "https://github.com/tensorflow/tensorflow/blob/f3b9bf4c3c0597563b289c0512e98d4ce81f886e/tensorflow/core/kernels/histogram_op.cc"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (histogram_op.cc#L35-L74)",
					"url": "https://github.com/tensorflow/tensorflow/blob/f3b9bf4c3c0597563b289c0512e98d4ce81f886e/tensorflow/core/kernels/histogram_op.cc#L35-L74"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (e57fd691c7b0fd00ea3bfe43444f30c1969748b5)",
					"url": "https://github.com/tensorflow/tensorflow/commit/e57fd691c7b0fd00ea3bfe43444f30c1969748b5"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (issues/45770)",
					"url": "https://github.com/tensorflow/tensorflow/issues/45770"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (v2.6.4)",
					"url": "https://github.com/tensorflow/tensorflow/releases/tag/v2.6.4"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (v2.7.2)",
					"url": "https://github.com/tensorflow/tensorflow/releases/tag/v2.7.2"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (v2.8.1)",
					"url": "https://github.com/tensorflow/tensorflow/releases/tag/v2.8.1"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (v2.9.0)",
					"url": "https://github.com/tensorflow/tensorflow/releases/tag/v2.9.0"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-20",
					"url": "https://jvndb.jvn.jp/ja/cwe/CWE-20.html",
					"title": "不適切な入力確認"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2023-08-14T17:25:58+09:00",
					"description": "[2023年08月14日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2023-08-14T17:25:58+09:00",
		"datelastupdated": "2023-08-14T17:25:58+09:00",
		"datepublic": "2022-05-21T00:00:00+09:00"
	}
}
